client/utils/gitUtils.js

```diff
@@ -0,0 +1,9 @@
+export const gitUpdatedDateToString = dateString => {
+    const date = new Date(dateString);
+    return `${date.toLocaleString('default', {
+        month: 'short'
+    })} ${date.getDate()}, ${date.getFullYear()} at ${date.toLocaleTimeString(
+        'default',
+        { timeZone: 'UTC' }
+    )}`;
```

Comment:
Is it possible to make the AM / PM lowercase? If there's no good way to do it, i.e. you have to use regex replace or something, then I guess I'd be happy with the uppercase version, but I do think the lowercase version seems more friendly.